### PR TITLE
Initial TSC version for the release procedure

### PR DESCRIPTION
**Pull request by balhar-jakub**
Signed-off-by: Jakub Balhar <jakub.balhar@broadcom.com>

### CODE REVIEW

**Comment by balhar-jakub** [comment]
Linked to #923

File: Technical-Steering-Committee/release.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+The vote is going to happen in the zowe-tsc channel as explained in the TSC Structure and TSC Charter. The option is to vote for releasing or for stopping the release. When the votes approves the release a TSC member, or the chair, will notify the release manager. The Release manager will then make the artifacts available and notify the OMP Director of the new release to track the LLC release of a new set of binaries.
+
+
+## Release Numbering
+
+Each release will be identified by a version number. These numbers are used according to a specific scheme that will give you additional information about the release.  The version numbers are of the form x.y.z-[GA | beta | yyyymmdd] (Semantic Versioning) or major.minor.micro. The final designation indicates whether this is the official Generally Available (GA) version, a beta version or an interim build. Only releases that end with **-GA** are considered official releases.
```
Comment (jackjia-ibm) [inline Technical-Steering-Committee/release.md:41]: `Only releases that end with **-GA** are considered official releases.` - In practice, we announce GA version without any suffix.

File: Technical-Steering-Committee/release.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  * MAJOR version when you make incompatible API changes
+  * MINOR version when you add functionality in a backwards-compatible manner, and 
+  * PATCH version when you make backwards-compatible bug fixes.
+
+* **GA** indicates that this release is an official, supported version of the Zowe project and is suitable for regular use.
+
+* **beta** means that this build is a candidate build that is complete but is under active development and should not be considered for regular use.
+
+* **yyyymmdd** indicates this is an interim build suitable for experimentation or development. It is not intended for general usage and may contain defects that are known and being worked or unknown.
+The code can be found[here](https://zowe.org/download/).
```
Comment (jackjia-ibm) [inline Technical-Steering-Committee/release.md:53]: In practice, we only announced GA version (without any suffix) and **RC?** versions. For RCs, it's not available directly on zowe.org/download, but can be found in Artifactory [libs-release-local/org/zowe/](https://zowe.jfrog.io/ui/repos/tree/General/libs-release-local%2Forg%2Fzowe). We also promote nightly builds in [libs-release-local/org/zowe/nightly/](https://zowe.jfrog.io/ui/repos/tree/General/libs-release-local%2Forg%2Fzowe%2Fnightly), but we didn't rename it. The Zowe night build name will look like `zowe-<version>-staging-<build-id>-<timestamp>.pax`.

We can reflect what we are doing, or adjust our pipelines to desired format.

File: Technical-Steering-Committee/release.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+## Release Content
+
+There are two significant and different release artifacts for Zowe. The source release and the "convenience build."
```
Comment (jackjia-ibm) [inline Technical-Steering-Committee/release.md:58]: We currently release these type of bundles which allows download from zowe.org:

- z/OS Component
  * Convenience Build
  * SMP/E Build
- CLI component (Also downloadable from npmjs.org)
  * CLI Core
  * CLI Plugins
- Client SDKs
  * Node.js Client SDK
  * Python Client SDK
- Zowe Explorer (Only downloadable from Visual Studio Code Marketplace)
- Docker build (Tech preview) (Also downloadable from hub.docker.com)
  * amd64
  * x390x

File: Technical-Steering-Committee/release.md (hunk 2 above)
Comment (balhar-jakub) [reply Technical-Steering-Committee/release.md:53]: I will update the text to reflect the current reality

File: Technical-Steering-Committee/release.md (hunk 3 above)
Comment (balhar-jakub) [reply Technical-Steering-Committee/release.md:58]: I will update the doc

**Pull request merged**